webpack/helpers/tableHelper.js

```diff
@@ -7,3 +7,11 @@ export const linkCell = (path, text) => (
     <Link to={path}>{text}</Link>
   </TableText>
 );
+
+export const hostPageLinkCell = (path, text, history) => (
```

Comment:
After some thinking, we don't need the `router` prop from the slot.
foreman's client routing uses `connected-react-router` so we can dispatch an action for triggering a url change.
we have already a helper for that under `foreman_navigation` (pushUrl)

Reply:
agree, the `pushUrl` helper  should be imported directly from `foremanReact`

Reply:
I mean it should be :) 
meanwhile, we can consume `push` directly from `connected-react-router` in plugins.

Reply:
I tried that, the following does not work as expected:

```js
import { push } from 'connected-react-router'
//
<a href="#" onClick={() => push(path)}>
  {text}
</a>
```